README.md

```diff
@@ -24,7 +24,9 @@ Includes protobuf and basic functions.
 
 ### Tools
 
-`cmd/localrpki` performs simple validation against files.
+`cmd/localrpki` performs simple validation against files and generate a JSON prefix list.
+
+`cmd/ctrpki` performs simple validation against files and send them to a Certificate Transparency Log.
```

Comment:
perhaps we could add a bit about this in the README file? Along with the flags. Also, perhaps we can improve the error messages, they are a bit confusing - example: 

```INFO[0000] Got open rsync://rpki.apnic.net/repository/apnic-rpki-root-iana-origin.cer: no such file or directory but repository not yet synchronized```